third_party/react-dates/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 ./node_modules/.bin/browserify \
--t [ babelify --presets env ] \
+-t [ babelify --presets [ @babel/preset-env ] ] \
```

Comment:
Did this break with the Babel 7 upgrade or was this build process using Babel 6?

Reply:
It was using the same babel version as the rest of AMP, so it needs to be updated. The script is run on-demand to generate a new bundle, not every build